README.md

```diff
@@ -9,10 +9,31 @@
 - [Dani Arribas-Bel](http://darribas.org/) -  University of Liverpool
 - [Joris Van den Bossche](https://jorisvandenbossche.github.io/) - Université Paris-Saclay Center for Data Science 
 
-
 This tutorial is an introduction to geospatial data analysis in Python, with a focus on tabular vector data. It is the first part in a series of two tutorials; this part focuses on introducing the participants to the different libraries to work with geospatial data and will cover munging geo-data and exploring relations over space. This includes importing data in different formats (e.g. shapefile, GeoJSON), visualizing, combining and tidying them up for analysis, and will use libraries such as `pandas`, `geopandas`, `shapely`, `PySAL`, or `rasterio`. The second part will built upon this and focus on more more advanced geographic data science and statistical methods to gain insight from the data. No previous experience with those geospatial python libraries is needed, but basic familiarity with geospatial data and concepts (shapefiles, vector vs raster data) and pandas will be helpful.
 
+## Outline of the Workshop
+
+- 7:50 - 8:20: **Installation & setup**
+
+  If you cannot complete the installation instructions ahead of time, please come slightly early so we can work on ensuring everyone can get the required packages installed and so that you can run the workshop material & exercises
+
+- 8:20 - 9:00: **Working with spatial data**
+
+  <!-- synthesize `01,04,05` -->
```

Comment:
I think 40 min is fine for the basic intro + exercises, but it seems a bit tight to also get the visualization stuff in there (notebooks 04/05)

Reply:
Maybe we can include some of that in Exploratory spatial data analysis ? (and if needed, give that part a bit more space in favor of the last part?)